src/main/java/com/google/sps/data/LearningPathService.java

```diff
@@ -77,20 +77,45 @@ public LearningPath load(long id) throws EntityNotFoundException {
 	}
 
 	private List<LearningSection> loadSections(long id) {
-		Query query = new Query(LEARNING_SECTION)
-				.addSort("sequence")
+		Query query = new Query(LEARNING_SECTION).addSort("sequence")
 				.setFilter(new Query.FilterPredicate("learningPath", Query.FilterOperator.EQUAL, id));
 
 		List<Entity> sections = datastore.prepare(query).asList(FetchOptions.Builder.withDefaults());
 
-		return sections.stream()
-				.map(e -> new LearningSection(e.getKey().getId(), (String) e.getProperty("name"),
-						"description", (long) e.getProperty("sequence")))
-				.collect(Collectors.toList());
+		return sections.stream().map(e -> {
+
+			LearningSection section = new LearningSection(e.getKey().getId(), (String) e.getProperty("name"),
+					"description", (long) e.getProperty("sequence"));
+
+			List<LearningItem> items = loadItems(section.getId());
+
+			section.getItems().addAll(items);
+
+			return section;
```

Comment:
I think there's enough code here that we should move this to a separate function that takes an Entity and creates a LearningSection.

`return sections.stream().map(this::createSection)....`